client/extensions/woocommerce/state/sites/setup-choices/actions.js

```diff
@@ -9,6 +9,8 @@ import { setError } from '../status/wc-api/actions';
 import {
 	WOOCOMMERCE_SETUP_CHOICES_REQUEST,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873